docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md

```diff
@@ -1547,3 +1547,60 @@ The following table provides a list of all source parameters and their default v
 | `email_notifications`	Email Notifications to be sent by the alert. | `[ ]` |
 | `connection_notifications` | Connection Notifications to be sent by the alert. | `[ ]` |
 | `parent_folder_id` | The folder ID is automatically generated. Do not enter a value for this parameter. This is the folder ID to install the apps into. A folder using the provided name will be added in "apps_folder_name". If the folder ID is empty, apps will be installed in the Personal folder. | Ignore this parameter. |
+
+## Troubleshooting
+
+This section provides information on how to troubleshoot failures while deploying AWS observability solution using Terraform.
+
+### Python command not found
+#### Error Message
+`python source-module/attach_fields_to_source.py`<br/>Python: command not found
```

Comment:
This is how the error message is displaying on localhost. Was this intentional?
<img width="300" alt="Screenshot 2023-05-17 at 2 06 50 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/92a553ec-af22-41d2-bd1c-d9349e81dad0">


Or should it be like this, all in one console message?
<img width="300" alt="Screenshot 2023-05-17 at 2 06 24 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/b163c156-3dbb-4a63-b1f1-c9b77a2a56c4">

Reply:
@kimsauce thanks for this. I feel its better we keep the error message in a console as you suggested.

Reply:
Fixed.